### PR TITLE
Update destinations.mdx

### PR DESCRIPTION
**Pull request by Matan2801**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx [hunk 1]
```diff
@@ -88,21 +88,9 @@ It's also possible to review the state using the Nerdgraph API ('status' propert
...
-<img
-  title="The filter tool that makes it easier to find the information you're looking for."
-  alt="A screenshot of the filter tool that makes it easier to find the information you're looking for."
-  src={accountsFilterDestinations}
-/>
-
-<figcaption>
-  Filter your destination logs by destination type, sent by, and status.
-</figcaption>
+The [Issue notification log](docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/#issue-notification-log) gives you a detailed record to keep track of all notifications generated by workflows. This information helps you troubleshoots issues, monitor workflow performance, and ensures that notifications are being sent to the correct destinations. As a result, you can quickly identify any issues that may arise and take corrective action as needed.
```
Comment (rhetoric101) [inline src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx:93]: @Matan2801 the link for "Issue notification log" seems to be broken. Could you double check it?

**Review by rhetoric101 — COMMENTED** [review]
@Matan2801 I had trouble with a link in this update. Could you check it?

**Review by nbaenam — DISMISSED** [review]
Thank you for the update!! It looks good to me!

File: src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx (hunk 1 above)
Comment (nbaenam) [reply src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx]: @rhetoric101 and @Matan2801, fixed! The URL must start with the "/"

**Review by nbaenam — APPROVED** [review]

File: src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/alerts-applied-intelligence/notifications/destinations.mdx]: @nbaenam thanks for your help with this!

**Pull request merged**